sdk/core/Azure.Core/tests/TestFramework/AsyncOnlyAttribute.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+namespace Azure.Core.Testing
+{
+
+    [AttributeUsage(AttributeTargets.Method|AttributeTargets.Class|AttributeTargets.Assembly, AllowMultiple=false, Inherited=false)]
+    public class AsyncOnlyAttribute : NUnitAttribute, IApplyToTest
+    {
+        #region IApplyToTest members
+
+        /// <summary>
+        /// Modifies a test by marking it as Ignored.
+        /// </summary>
+        /// <param name="test">The test to modify</param>
+        public void ApplyToTest(Test test)
+        {
+            if (test.RunState != RunState.NotRunnable)
+            {
+                // This is an unfortunate implementation but it's the only one I was able to figure out
+                string testParameters = test.FullName.Substring(test.ClassName.Length);
+                if (testParameters.StartsWith("(False"))
```

Comment:
Worth filing a bug to dig into further?

Reply:
Not sure there's a lot to dig into, this information doesn't seem to be on the test object, unfortunately. Let's see if it causes any issues like this.